site/src/pages/RoadmapPage/transfers/APExamsSection.tsx

```diff
@@ -1,17 +1,42 @@
-import { FC, useState } from 'react';
+import { FC, useState, useContext, useEffect, useCallback } from 'react';
 import MenuSection, { SectionDescription } from './MenuSection';
 import MenuTile from './MenuTile';
+import Select from 'react-select';
+import ThemeContext from '../../../style/theme-context';
+import { comboboxTheme } from '../../../helpers/courseRequirements';
+import trpc from '../../../trpc';
+import { useAppDispatch } from '../../../store/hooks';
 
 interface ScoreSelectionProps {
   score: number | null;
   setScore: (value: number) => void;
 }
 
+interface APExam {
+  fullName: string;
+  catalogueName: string;
+  rewards: {
+    acceptableScores: number[];
+    unitsGranted: number;
+    electiveUnitsGranted: number;
+    geCategories: string[];
+    coursesGranted: {
+      string: string[];
+    };
+  }[];
+}
+
```

Comment:
Looks like you've redefined the same interface. I know it's still a draft PR, but I would suggest in that case creating `transferCredits.ts` in `types/src/` and adding those two lines from the previous comment